stock_partner_lot/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=======================================
+Show lots on the partners that own them
+=======================================
+
+This module lets users view which lots have been transferred to each partner and
+search accordingly.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
Please include the section `Usage` with instructions on how to get a "lot with owner" (I suppose that it's a lot moved to a location that has an owner on it).